io.itch.doctorm64.skippybot.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Categories=Game
+Encoding=UTF-8
+Name=Skippy The Bot
+Comment=Skippy is on an epic quest to unravel its true destiny.
+Exec=/app/bin/skippybot
```

Comment:
```suggestion
Exec=skippybot
```

Reply:
That won't work. The Unity engine (which this game uses) is a bit special in how it wants games to be launched. It needs to be launched via the absolute path, as otherwise the game will not find it's assets.
I meant to add a comment to this before pushing, but forgot. Added it now locally.